### ISSUE TITLE
Wiki images

### ISSUE THREAD

**Issue by randyxli** [randyxli/mplant#1]
![map](https://cloud.githubusercontent.com/assets/5498027/15236868/0969f104-1897-11e6-949f-bba15de4c5e9.png)

![notes](https://cloud.githubusercontent.com/assets/5498027/15237090/e08d1d4a-1898-11e6-98d8-df3600044fd5.png)

![screen shot 2016-05-13 at 12 38 51 pm](https://cloud.githubusercontent.com/assets/5498027/15255047/ca29cf8e-1907-11e6-85b5-564d09b441ea.png)

Things to include in the app

Info that users would have to input
1. General info about the apartment listing --> ie sq ft, neighborhood, kitchen?, whole apt or private room? private bathroom? Is it furnished?
2. Work address (to find the commute times on the map)
3. Furniture / Appliances / Utilities --> ie oven, dishwasher, wireless internet, TV etc.
4. Photos
5. Price (multiple prices if there are options)

Info that the app would collect from third party sites
1. Neighborhood info of the listing (general atmosphere, transportation)
2. Standard price of a 1bed, 2bed, etc listing in the area.

**Comment by cjunmokim** [comment]
I think we should think about how users will find listings to begin with, or should the app act exclusively as a organization tool with input data?

^ Actually I think we should keep it as an organizational tool. Users would go to CL or other third party websites to find listings. If not, we would have to import data from third party websites but would not be able to offer similarly robust functions, so users would not really use our app for this.

**Comment by randyxli** [comment]
Yeah agreed. It would be nice to have a data import tool though, where users can paste a link from Craigslist and the tool grabs some standard fields. Lets move our discussion to our requirements page on the Wiki maybe? https://github.com/randyxli/mplant/wiki/Requirements

edit: looks like there's no comment function on wikis. I've created a new issue to discuss requirements, to leave this one as an image hosting issue: https://github.com/randyxli/mplant/issues/2
